src/neuroconv/tools/spikeinterface/spikeinterface.py

```diff
@@ -1464,8 +1468,8 @@ def add_waveforms(
     )
 
 
-def write_waveforms(
```

Comment:
Can you keep these functions with the previous signature and they  throw and informative error of when was the last version of neuroconv where they were supported if they are called? I don't want to throw an import error without context if someone is relying on this.